libs/config/table-config/src/lib/table-config.service.ts

```diff
@@ -17,7 +21,7 @@ export class TableConfigService {
       return pref;
     }
 
-    return 10;
+    return this.defaultTableSizes.get(tableId) || 10;
```

Comment:
Personally I would prefer this over the `||` operator just to prevent potential confusion. Not sure if it's some javascript special but I assumed it would return a boolean and that it's a mistake.
```suggestion
    return this.defaultTableSizes.get(tableId) ?? 10;
```